service/kbupd/src/bin/kbupctl.rs

```diff
@@ -73,7 +71,11 @@ fn main() -> Result<(), failure::Error> {
                 })
                 .map(move |(reply, _framed): (ControlReply, ControlFramed)| {
                     if let Some(control_reply::Data::GetStatusControlReply(reply)) = reply.data {
-                        print_fun(enclave_name, reply);
+                        match subcommand_name.as_str() {
+                            "info" => print_info(enclave_name, reply),
```

Comment:
Optional: It seems useful and more consistent for the `info` command to also support JSON